packages/ubuntu_init/lib/src/launchsession/launchsession_model.dart

```diff
@@ -0,0 +1,14 @@
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+final launchsessionModelProvider = ChangeNotifierProvider(
+  (ref) => LaunchsessionModel(),
+);
+
+class LaunchsessionModel extends ChangeNotifier {
+  LaunchsessionModel();
+
+  Future<bool> init() async => true;
+}
```

Comment:
See comment above - let's create a method here that fetches the identity and calls your service's `openNewSession`.

Reply:
Done